### PR TITLE
[bugfix](workloadgroup) could not upgrade from 2.0 alpha

### PR DESCRIPTION
**Pull request by yiguolei**
## Proposed changes

2.0 alpha the workload group does not have property mem_limit, but it is a required propery in 2.0 beta, so the check will failed.

Skip the check and allow upgrade successfully.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by wangbo — DISMISSED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Review by xiaokang — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroup.java [hunk 1]
```diff
@@ -281,8 +281,13 @@ public static WorkloadGroup read(DataInput in) throws IOException {
...
     @Override
     public void gsonPostProcess() throws IOException {
-        String memoryLimitString = properties.get(MEMORY_LIMIT);
-        this.memoryLimitPercent = Double.parseDouble(memoryLimitString.substring(0, memoryLimitString.length() - 1));
+        if (properties.containsKey(MEMORY_LIMIT)) {
+            String memoryLimitString = properties.get(MEMORY_LIMIT);
+            this.memoryLimitPercent = Double.parseDouble(memoryLimitString.substring(0,
+                    memoryLimitString.length() - 1));
+        } else {
+            this.memoryLimitPercent = 100;
```
Comment (luozenglin) [inline fe/fe-core/src/main/java/org/apache/doris/resource/workloadgroup/WorkloadGroup.java:289]: properties.put(MEMORY_LIMIT, "100");
this.memoryLimitPercent = 100;

**Comment by yiguolei** [comment]
run buildall

**Comment by yiguolei** [comment]
run buildall

**Review by luozenglin — APPROVED** [review]
LGTM

**Review by wangbo — APPROVED** [review]
LGTM

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 44.78 seconds
 stream load tsv:          460 seconds loaded 74807831229 Bytes, about 155 MB/s
 stream load json:         21 seconds loaded 2358488459 Bytes, about 107 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 insert into select:          68.9 seconds inserted 10000000 Rows, about 145K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230625132154_clickbench_pr_167244.html

**Pull request merged**